src/Hydra/Protocol/OnChain.tex

```diff
@@ -389,169 +505,167 @@ \subsection{Decrement Transaction}\label{sec:decrement-tx}
 used snapshot number and $m$ is the number of outputs to distribute. The
 validator checks:
 \begin{menumerate}
-	\item State is advanced from $\datumHead \sim \stOpen$ to
-	$\datumHead' \sim \stOpen$, parameters $\cid,\hydraKeys,\cPer$ stay
-	unchanged and the new state is governed again by $\nuHead$
-	\[
-		(\stOpen,\cid,\hydraKeys,\cPer,v,\eta) \xrightarrow[\xi, s, m]{\mathsf{decrement}} (\stOpen,\cid,\hydraKeys,\cPer,v',\eta')
-	\]
-	\item New version $v'$ is incremented correctly
-	\[
-		v' = v + 1
-	\]
-	\item $\xi$ is a valid multi-signature of the new snapshot state (currency id $\cid$, the current state version $v$, the new state $\eta'$ \red{, UTxO to deposit $\eta_\alpha$} and UTxO to withdraw $\eta_\omega$)
-	\[
-		\msVfy(\hydraKeys,(\cid || v || s || \eta' || \red{\bot} || \eta_\omega),\xi) = \true
-	\]
-    where snapshot number $s$ is provided through the redeemer and $\eta_\omega$ must be the digest of all removed UTxO.
-	\[
-		\red{\eta_\omega} = \hash(\bigoplus_{j=2}^{m+1} \bytes(o_{j}))
-	\]
-	\item The value in the head output is decreased accordingly
-	\todo{Redundant to above? Only check $\valHead' < \valHead$?}
-	\[
-		\valHead' \cup (\bigcup_{j=2}^{m+1} \val_{o_{j}}) = \valHead
-	\]
-	\item Transaction is signed by a participant
-	\todo{Allow anyone to do decrements?}
-	\[
-		\exists \{\cid \mapsto \keyHash_{i} \mapsto 1\} \in \valHead' \Rightarrow \keyHash_{i} \in \txKeys
-	\]
+  \item State is advanced from $\datumHead \sim \stOpen$ to
+  $\datumHead' \sim \stOpen$, parameters $\cid,\hydraKeys,\cPer$ stay
+  unchanged and the new state is governed again by $\nuHead$
+  \[
+	(\stOpen,\cid,\hydraKeys,\cPer,v,\eta) \xrightarrow[\xi, s, m]{\mathsf{decrement}} (\stOpen,\cid,\hydraKeys,\cPer,v',\eta')
+  \]
+  \item New version $v'$ is incremented correctly
+  \[
+	v' = v + 1
+  \]
+  \item $\xi$ is a valid multi-signature of the new snapshot state $\eta'$
+  \[
+	\msVfy(\hydraKeys,(\cid || v || s || \eta' || \red{\bot} || \eta_\omega),\xi) = \true
+  \]
+  where $\eta_\omega$ must is the digest of all removed UTxO
```

Comment:
typo here

Reply:
Fixed in https://github.com/cardano-scaling/hydra-formal-specification/commit/2dcc790e4cb4569d684ab73c093939af98461a03